test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -41,6 +41,9 @@
 from torch.testing._internal import common_utils
 from torch.testing._internal.common_utils import skipIfNoLapack
 
+# The max onnx opset version to test for
+MAX_ONNX_OPSET_VERSION = 17
```

Comment:
How about using `torch/onnx/_constants.py`'s `onnx_main_opset` instead of duplicating this information?

Reply:
I intentionally did this to isolate values in _constants from those in test. So changing _constants will not affect the tests run. Now that I look at it again, this seems a good idea to make sure an incremented max version also triggers new set of tests to run. @BowenBao do you have any thoughts?

Reply:
(Because we want tests to _break_ when changes happen, but isolating `MAX_ONNX_OPSET_VERSION` does not cause tests to break when new opsets are added)

Reply:
> I intentionally did this to isolate values in _constants from those in test. So changing _constants will not affect the tests run. Now that I look at it again, this seems a good idea to make sure an incremented max version also triggers new set of tests to run. @BowenBao do you have any thoughts?

Ideally, all tests should reflect any change to the actual source code change, otherwise tests are not covering everything they should

Keeping opset version for tests disconnected from the opset version for the converter seems dangerous to me. We may forget about changing the test constant and assume everything is being teste and working because CI is green.

Reply:
I agree. I am leaning toward using the max number from _constants and keeping the min number separate

Reply:
Done